protocol-definitions/Metrics.yaml

```diff
@@ -0,0 +1,35 @@
+id: 40
+name: Metrics
+methods:
+  - id: 1
+    name: readMetrics
+    since: 2.0
+    doc: |
+      Reads the recorded metrics starting with the smallest sequence number
+      greater or equals to the sequence number set in fromSequence.
+    request:
+      retryable: false
+      acquiresResource: false
+      partitionIdentifier: name
+      params:
+        - name: uuid
+          type: String
+          nullable: false
+          since: 2.0
+          doc: |
+            The UUID of the member that is supposed to read the metrics from.
+        - name: fromSequence
+          type: long
+          nullable: false
+          since: 2.0
+          doc: |
+            The sequence the recorded metrics should be read starting with.
+    response:
+      params:
+        - name: response
+          type: Data
```

Comment:
what is the type of this Data? Can we expose it from protocol point of view? if it's just a simple class, you can list the parameters instead of reponse param or create a custom type ?

Reply:
It may send either an exception, or a `com.hazelcast.internal.metrics.managementcenter.ConcurrentArrayRingbuffer.RingbufferSlice`. The latter is added in https://github.com/hazelcast/hazelcast/pull/15560/files#diff-f3e47ace2dc339bf108b5b8c1fb5d15eR126.